.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+# Description
+
+Please include a summary of the changes and features. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+# URL to relevant Jira ticket
+
+Please include the link to the relevant Jira ticket here for ease of cross-referencing.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Technical Enhancements
+
+
+# Checklist:
+
+If any of these are not completed, please explain why in notes.
+
+## **Definition of Done**
+**Code and merges**
+
+- [ ] Code review (informal/walk through (and/or produced with Pair programming)
+- [ ] Code review completed by the appropriate people (1 thumbs up)
```

Comment:
is it worth getting rid of the review requirements? its annoying having to go in and edit the body once its been done and we cant merge changes until its been reviewed by someone with the correct privileges anyway

Reply:
I am happy to remove this if everyone else is :)

Reply:
@gibbardsteve  opinon? :)

Reply:
Sorry one last thing: Could you add a checkbox to remind people to add a link to the PR in the JIRA ticket. It would be good to get in the habit of updating the tickets in JIRA with a link to the PR. (I know I am an admin monster, but it is useful having the backward linking and a good habit to get into).

Reply:
On it :)

Reply:
no worries Steve better to get this all sorted now then we no longer have to worry about admin :)

Reply:
@gibbardsteve and @JasonBellONS is this now okay to merge?

Reply:
We removed the unit test checkbox as sml catalog does not have unit tests :)